core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/NameRepairStrategy.kt

```diff
@@ -0,0 +1,16 @@
+package org.jetbrains.kotlinx.dataframe.io
+
+/**
+ * This strategy defines how the repeatable name column will be handled
+ * during the creation new dataframe from the IO sources.
+ */
+public enum class NameRepairStrategy {
```

Comment:
It's unclear that it targets duplicate names just by looking at the enum class name.  Maybe call it `DuplicateNameStrategy`?

Reply:
In many dataframe frameworks it's named as NameRepairing, better to keep naming to reduce learning curve

Reply:
I think "repairing" insinuates a name is "broken", e.g. unsupported characters etc. This is definitely different IMO. Can you give an example from another library?

Reply:
https://readr.tidyverse.org/reference/read_delim.html#arguments
go to the name_repair

Reply:
I see! Well, if in the future we also add "universal", which fixes name syntax too, I'm fine with the name. Then it won't be just about uniqueness